src/apps/backend/modules/otp/internal/store/otp-db.ts

```diff
@@ -0,0 +1,44 @@
+import { Schema, Types } from 'mongoose';
+
+import { PhoneNumber } from '../../../account';
+import { OtpStatus } from '../../types';
+
+export interface OtpDB {
+  _id: Types.ObjectId;
+  active: boolean;
+  otpCode: string;
+  phoneNumber: PhoneNumber;
+  status: OtpStatus;
+}
+
+export const OtpDbSchema: Schema = new Schema<OtpDB>(
```

Comment:
don't you need a combined index of phone and active as I see you are using that in your query pattern?